deckhouse-controller/cmd/deckhouse-controller/register-go-hooks.go

```diff
@@ -42,6 +42,7 @@ import (
 	_ "github.com/deckhouse/deckhouse/modules/300-prometheus/hooks"
 	_ "github.com/deckhouse/deckhouse/modules/301-prometheus-metrics-adapter/hooks"
 	_ "github.com/deckhouse/deckhouse/modules/302-vertical-pod-autoscaler/hooks"
+	_ "github.com/deckhouse/deckhouse/modules/340-automatic-namespaces-discovery/hooks"
```

Comment:
Let's give it the 600 number prefix (like the secret-copier module), and move it to ee/fe folder.